engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/ExternalTaskRestService.java

```diff
@@ -32,11 +32,7 @@
 
 import org.camunda.bpm.engine.rest.dto.CountResultDto;
 import org.camunda.bpm.engine.rest.dto.batch.BatchDto;
-import org.camunda.bpm.engine.rest.dto.externaltask.ExternalTaskDto;
-import org.camunda.bpm.engine.rest.dto.externaltask.ExternalTaskQueryDto;
-import org.camunda.bpm.engine.rest.dto.externaltask.FetchExternalTasksDto;
-import org.camunda.bpm.engine.rest.dto.externaltask.LockedExternalTaskDto;
-import org.camunda.bpm.engine.rest.dto.externaltask.SetRetriesForExternalTasksDto;
+import org.camunda.bpm.engine.rest.dto.externaltask.*;
```

Comment:
Star imports are not allowed according to our style guide. See [Coding Style Java Imports](https://github.com/camunda/camunda-bpm-dev-docs/blob/master/development/Coding-Style-Java.md#imports).